docs/source/docs/installation/installation_dev.rst

```diff
@@ -8,6 +8,7 @@ You will need:
 
 -  `node`_
 -  `Python 3`_
+-  `Poetry`_
```

Comment:
This is current an invalid link. Look into the RST format to learn how to make this hyperlink.

Reply:
This is now the right syntax, but you need to link to the page that describes how to install poetry. Look at the other links for inspiration.

Reply:
Use this URL: https://python-poetry.org/docs#installation